src/components/molecules/TodoActions.vue

```diff
@@ -7,39 +7,74 @@ const store = useStore()
 
 const filters = computed(() => store.getters.getFilters)
 const remaining = computed(() => store.getters.undoneTodosCount)
+const visibility = computed(() => store.getters.getVisibility)
```

Comment:
I think ''visisbility'' doesn't quite show what the variable/method is responsible for. Maybe change the name to something more suitable for the filtering logic (activeFilter, filteredItems ...)?